packages/core/contracts/financial-templates/perpetual-multiparty/ConfigStore.sol

```diff
@@ -153,24 +161,34 @@ contract ConfigStore is ConfigStoreInterface, Testable, Lockable, Ownable {
 
     // Use this method to constrain values with which you can set ConfigSettings.
     function _validateConfig(ConfigStoreInterface.ConfigSettings memory config) internal pure {
-        // Its hard to ascertain what are reasonable limits, into the future and the past, for proposal timestamps.
-        // So, we do not set any limits.
+        // We don't set limits on proposal timestamps because there are already natural limits:
+        // - Future: price requests to the OptimisticOracle must be in the past (plus some buffer allowing for the time
```

Comment:
@mrice32 The OO doesn't currently enforce requests to be in the past, I can wait to. merge this until that PR is done.

Reply:
Sounds good. Will put up a PR momentarily.